primitives/session-keys/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = ['Manta Network']
+description = "Primitives for session keys"
+edition = "2021"
+license = 'GPL-3.0'
+name = "session-key-primitives"
+version = "1.0.0"
+
+[dependencies]
+manta-primitives = { path = "../manta", default-features = false }
+nimbus-primitives = { git = "https://github.com/manta-network/nimbus.git", branch = "polkadot-v0.9.22", default-features = false }
+parity-scale-codec = { version = "3.0.0", default-features = false, features = ["derive"] }
```

Comment:
`codec` and `scale-info` should be bumped, please take a look at other crates.
It should be `2.1.2` and `3.1.2`.